components/AuthForm/AuthForm.tsx

```diff
@@ -4,6 +4,8 @@ import { useRouter } from "next/router";
 import { Layout } from "../Layout";
 import { Formik, Form, Field, ErrorMessage } from "formik";
 import styled from "@emotion/styled";
+import { SignupForm } from "../SignupForm";
+
 import {
   AuthFormType,
   loginForm,
```

Comment:
The line below this one: "signupForm" is still imported but not used, let's remove this one ...